README.md

```diff
@@ -42,6 +42,9 @@ cd web/profiles/custom/wilson/themes/wilson_theme_starterkit
 npm install && gulp build
 drush cr
 ```
+The theme also provides Storybook integration by default. [Read more about Storybook with Drupal](https://medium.com/@askibinski/integrating-storybook-with-drupal-ddabfc6c2f9d).
```

Comment:
This might be better placed in the README of the theme itself. But for the purposes of the Wilson home page we can mention the theme being Storybook enabled.